### PR TITLE
feat: image block updates

### PR DESCRIPTION
**Pull request by SamCreasey**
CU-8693kyqyc

Initial design changes for the image block:
- Moved "alt text" from the flyout menu into a separate component
- Added delete option but kept existing menu item

### CODE REVIEW

**Comment by ragi96** [comment]
merged so somebody gets assigned, sorry didn't had it in my mind

**Review by utmishra — CHANGES_REQUESTED** [review]
Hey @SamCreasey 
Visually, the changes look good 👍🏽 
There seems to be just one issue. Once I've set the alt text, I cannot remove it. If I delete the existing entry and save the alt text with blank text, I see that an empty `settings` payload is passed, thus the alt text is not updated.

File: packages/image-block/src/ImageBlock.tsx [hunk 1]
```diff
@@ -44,7 +44,6 @@ export const ImageBlock = withAttachmentsProvider(({ appBridge }: BlockProps) =>
     const { openAssetChooser, closeAssetChooser } = useAssetChooser(appBridge);
     const isEditing = useEditorState(appBridge);
     const blockId = appBridge.getBlockId().toString();
```
Comment (ragi96) [inline packages/image-block/src/ImageBlock.tsx:46]:
```suggestion
    const blockId = String(appBridge.context('blockId').get());
```

File: packages/shared/src/components/EditAltTextFlyout/EditAltTextToolbarButton.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (ragi96) [inline packages/shared/src/components/EditAltTextFlyout/EditAltTextToolbarButton.tsx:1]: Would be cool to have this one tested

Comment (SamCreasey) [reply packages/shared/src/components/EditAltTextFlyout/EditAltTextToolbarButton.tsx:1]: fixed

**Review by ragi96 — APPROVED** [review]
Next time please in `vitest`, as we will migrate them anyway

**Review by utmishra — APPROVED** [review]
Looks good 👍🏽

**Pull request merged**